packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -287,7 +257,7 @@ const EDiscovery = WebexPlugin.extend({
 
     // If the content summary for this space has already been cached then it can be retrieved from there instead of making a network call to ediscovery
     if (this.contentSummaryBySpaceIdCache.has(reportId) && this.contentSummaryBySpaceIdCache.get(reportId).has(spaceId)) {
-      return this.contentSummaryBySpaceIdCache.get(reportId).get(spaceId);
+      return {body: this.contentSummaryBySpaceIdCache.get(reportId).get(spaceId), statusCode: 200};
```

Comment:
A response from the cache should mirror a response from the server

Reply:
Does the cache ever expire, or need to?

Reply:
Right now it lasts for as long as the downloader client is open, it shouldn't be a problem for now, but we may have to come back and add a cleanup mechanism at some point.